api/pom.xml

```diff
@@ -404,6 +404,8 @@ Use is subject to <a href="{@docRoot}/doc-files/speclicense.html" target="_top">
                 <configuration>
                     <forkCount>2</forkCount>
                     <reuseForks>false</reuseForks>
+                    <!-- Test service loader is not loaded if modules are enabled -->
+                    <useModulePath>false</useModulePath>
```

Comment:
for this to work, tests need their own module-info, there are more ways to make it work

Reply:
Added, thank you.